specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/virtualWan.json

```diff
@@ -5448,6 +5448,240 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/routingIntent/{routingIntentName}": {
```

Comment:
resource types should be plural, "routingIntents"

Reply:
this resource is meant to be a singleton resource which is why it is not plural